antlr/src/main/antlr/FSHLexer.g4

```diff
@@ -62,10 +62,10 @@ ARROW:              '->';
 // PATTERNS
 
                  //  "    CHARS    "
-STRING:             '"' (~[\\"] | '\\r' | '\\n' | '\\t' | '\\"' | '\\\\')* '"';
+STRING:             '"' (~[\\"] | '\\u' | '\\r' | '\\n' | '\\t' | '\\"' | '\\\\')* '"';
 
                  //  """ CHARS """
-MULTILINE_STRING:   '"""' .*? '"""';
+MULTILINE_STRING:   '"""' (. | '\\u')*? '"""';
```

Comment:
Is this necessary? I reverted it back to `.*?`, rebuilt the grammar, and re-ran the tests -- and they all passed.

Reply:
Hm I thought so, but I believe you, so I'll try it out again.

Reply:
I thought I had hit a case with a parser error, but I haven't found that case again, so it must have been something different. Reverted this change in [f56425a](https://github.com/FHIR/sushi/pull/1233/commits/f56425a39cb4ca0491dd7bf630206391622956bd).